testcases/MachineConfig.py

```diff
@@ -158,10 +161,19 @@ def callConfig(self, key, lpar=""):
                 num_hugepages = re.findall(
                     'hugepages=[0-9]+',
                     str(self.machine_config))[0].split('=')[1]
+            if "iocapacity" in config_value:
+                setup=1
+                if self.bmc_type == "EBMC_PHYP":
+                    ioenlargecapacity = re.findall(
+                    'iocapacity=[0-9]+', str(self.machine_config))[0].split('=')[1]
+                elif self.bmc_type == "FSP_PHYP:"
+                    ioenlargecapacity = re.findall(
+                    'iocapacity=[0-9]+', str(self.machine_config))[0].split('=')[1]
```

Comment:
why if else needed as in both case logic same

Reply:
Sure, Will update it as suggested